server/plugins/inputs/index.ts

```diff
@@ -89,9 +101,22 @@ const getCommands = () =>
 		.concat(passThroughCommands)
 		.sort();
 
-const addPluginCommand = (packageInfo: PackageInfo, command, func) => {
-	func.packageInfo = packageInfo;
-	pluginCommands.set(command, func);
+const addPluginCommand = (packageInfo: PackageInfo, command: any, obj: any) => {
```

Comment:
should `obj: any` be `obj: ExternalPluginCommand`?

Reply:
nope, it's provided by an external package, we literally don't know what it is, we must kot lie to the compiler / human

Reply:
nit: `unknown` is preferred then

Reply:
ack, will change

Reply:
actually... if I do that the typing gets really annoying.

It errors out even if I guard that this is an object, due to it not having the input property.
Now, I can either type assert it or just keep it as any.

Type asserting it seems worse to me, opinons?

Reply:
Then I think its fine as is. Sorry for wasting your time